examples/pipeline/actor/pedestrian_detection.h

```diff
@@ -0,0 +1,236 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * AS IS BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+ * Copyright (c) 2021
+ * Author: tpoisonooo
+ */
+#pragma once
+#include "../graph/node.h"
+#include "tengine/c_api.h"
+#include <cstdio>
+#include <cstdlib>
+#include <memory>
+#include <opencv2/opencv.hpp>
+#include <sys/time.h>
+
+namespace pipe {
+
+struct Box_t {
+  int x0;
+  int y0;
+  int x1;
+  int y1;
+  int class_idx;
+  float score;
+};
+
+class PedestrianDetection : public Node<Param<cv::Mat>, Param<std::tuple<cv::Mat, cv::Rect>>> {
+public:
+  PedestrianDetection(std::string model_path) {
+    /* inital tengine */
+    init_tengine();
+    fprintf(stderr, "tengine-lite library version: %s\n",
+            get_tengine_version());
+    m_graph = create_graph(NULL, "tengine", model_path.c_str());
```

Comment:
use context_t to adapt other backends
such as
```
m_ctx = create_context("ctx", 1);
auto ret = set_context_device("dev", m_dev, NULL, 0);
if (0 != ret)
{
    // just a warning
}
m_graph = create_graph(NULL, "tengine", model_path.c_str());
```

and m_dev should be inited firstly.